### PR TITLE
add marker support

### PR DESCRIPTION
**Pull request by gabriele83**

### CODE REVIEW

File: src/main/java/io/logz/logback/LogzioSender.java [hunk 1]
```diff
@@ -129,9 +130,9 @@ private LogzioSender(String logzioToken, String logzioType, int drainTimeout, in
     }
 
     public static synchronized LogzioSender getOrCreateSenderByType(String logzioToken, String logzioType, int drainTimeout, int fsPercentThreshold, String bufferDir,
-                                                       String logzioUrl, int socketTimeout, int connectTimeout, boolean debug,
-                                                       LogzioLogbackAppender.StatusReporter reporter, ScheduledExecutorService tasksExecutor,
-                                                       boolean addHostname, String additionalFields, int gcPersistedQueueFilesIntervalSeconds) {
+                                                                    String logzioUrl, int socketTimeout, int connectTimeout, boolean debug,
```
Comment (asafm) [inline src/main/java/io/logz/logback/LogzioSender.java]: If only spaces added, let's revert to the original indentation please, so we can focus on what's been added

File: src/main/java/io/logz/logback/LogzioSender.java [hunk 2]
```diff
@@ -62,9 +63,9 @@
     private final AtomicBoolean drainRunning = new AtomicBoolean(false);
 
     private LogzioSender(String logzioToken, String logzioType, int drainTimeout, int fsPercentThreshold, String bufferDir,
-                        String logzioUrl, int socketTimeout, int connectTimeout, boolean debug,
-                        LogzioLogbackAppender.StatusReporter reporter, ScheduledExecutorService tasksExecutor,
-                        boolean addHostname, String additionalFields, int gcPersistedQueueFilesIntervalSeconds)
+                         String logzioUrl, int socketTimeout, int connectTimeout, boolean debug,
```
Comment (asafm) [inline src/main/java/io/logz/logback/LogzioSender.java]: If only spaces added, let's revert to the original indentation please, so we can focus on what's been added

**Comment by gabriele83** [comment]
Ok

Il 20 nov 2016 09:34, "Asaf Mesika" <notifications@github.com> ha scritto:

**Comment by gabriele83** [comment]
news?

**Comment by gabriele83** [comment]
Thank you

Il 21 nov 2016 20:15, "Asaf Mesika" <notifications@github.com> ha scritto:

**Comment by gabriele83** [comment]
when it will be updated on the maven central?

**Comment by roiravhon** [comment]
@gabriele83 just pushed, can take up to 2 hours to propagate.
Thanks for that PR!

**Comment by gabriele83** [comment]
Tks

Il 22 nov 2016 09:56, "Roi Rav-Hon" <notifications@github.com> ha scritto:

**Pull request merged**